robot-assistant/README.md

```diff
@@ -0,0 +1,15 @@
+## Build Your Own Custom Robot Assistant
+
+This example remakes this project described in this blog https://www.fluvio.io/blog/2021/01/bot-assistant/ in Rust. It depends on the fluvio [Rust API](https://docs.rs/fluvio/0.4.0/fluvio/) instread of fluvio [Node.js API](https://www.npmjs.com/package/@fluvio/client).
+
+- The html static assets is in the html folder.
+- The tide websocket server is server folder.
+- The wasm client is in wasm folder.
+- The wasm pkg will be built in pkg folder.
+- Makefile.toml helps build the project.
+
+Steps to start the project
+
+- Cargo make robot-assistant-topic
+- Cargo make robot-assistant-build
```

Comment:
problem with build:
```
cargo make robot-assistant-build
[cargo-make] INFO - cargo make 0.32.12
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: robot-assistant-build
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: robot-assistant-server
[cargo-make] INFO - Running Task: robot-assistant-wasm
/tmp/ubuntu/fsio/RbdAhb2vzB.sh: 3: /tmp/ubuntu/fsio/RbdAhb2vzB.sh: wasm-pack: not found
[cargo-make] ERROR - Error while executing command, exit code: 127
[cargo-make] WARN - Build Failed.
ubuntu@ip-172-31-20-82:~/fluvio-demo-apps-rust/robot-assistant$ warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /home/ubuntu/fluvio-demo-apps-rust/robot-assistant/wasm/Cargo.toml
workspace: /home/ubuntu/fluvio-demo-apps-rust/Cargo.toml

```

Reply:
I need to add wasm-pack instruction here.